modmesh/app/euler1d.py

```diff
@@ -78,19 +79,43 @@ def __init__(self, figsize=(15, 10)):
         self.ax.set_ylim(-0.2, 1.2)
         self.ax.grid()
 
-        self.density = QuantityLine(name="density")
-        self.velocity = QuantityLine(name="velocity")
-        self.pressure = QuantityLine(name="pressure")
+        self.density = QuantityLine(name="density", unit="kg/m^3")
+        self.velocity = QuantityLine(name="velocity", unit="m/s")
+        self.pressure = QuantityLine(name="pressure", unit="pa")
 
     def build_lines(self, x):
         self.ax.set_xlim(x[0], x[-1])
-        for data, color in (
+        lines = []
+        for i, (data, color) in enumerate((
                 (self.density, 'r'),
                 (self.velocity, 'g'),
                 (self.pressure, 'b'),
-        ):
-            data.ana, = self.ax.plot(x.copy(), np.zeros_like(x), f'{color}-')
-            data.num, = self.ax.plot(x.copy(), np.zeros_like(x), f'{color}x')
+        )):
+            if i == 0:
```

Comment:
It's more readable to check for `isinstance(data, self.density)`.  In that way you can freely change the order of the data lines in the future.

Reply:
Checking `isinstance(data, self.density)` it will force `density` plot on main axis, therefore I can not freely change the order of the data lines.
Due to plot multiple data line on same x axis, it need plot a data line on main axis, therefore I check for loop index `I == 0` to plot data line on main axis, others plot on auxiliary axis.

Reply:
So you want the first data but rather than density to be plot as the main line.  That makes sense.